packages/webapp/public/locales/fr/translation.json

```diff
@@ -470,8 +470,8 @@
     "SATURDAY": "S",
     "SUNDAY": "D",
     "THURSDAY": "J",
-    "TUESDAY": "Ma",
-    "WEDNESDAY": "Me"
+    "TUESDAY": "M",
```

Comment:
@kathyavini Was wondering if we should leave these as they were actually. While understandable and our choice the rcYearCalendar defaults seems to use Ma and Me. And the English actually uses two letter codes instead of one letter.

Whats your opinion on if we should align all these as best we can or leave it ?

Reply:
> And the English actually uses two letter codes instead of one letter in that one.

Yeah, in the calendar English uses two letter codes for all of them though, which looks good.

I think the sometimes one-letter and sometimes two-letter code in French doesn't look very good (definitely in the repeat component but even in the calendar, really). Since we got Loïc's permission to go one-letter, I think I'd like to keep it the way he reviewed it. I don't think alignment is necessary since they both make sense in their respective contexts.